orbit/CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+## Orbit 1.28.0 (Jul 18, 2024)
+
+* Hide "Self-service" in Fleet Desktop and My device page if there is no self-service software available.
+
+* Fixes a bug that caused log Orbit's osquery table log output to be inconsistent.
+
+* Adding support for new agent option `script_execution_timeout` to configure seconds until a script is killed due to timeout.
+
+* Updated Go version to go1.22.4
+
+* Fix boot loop caused by Linux hosts with no hardware UUID
+
+* Added support for Linux ARM64
+
+* Fix bug where UTC timezone could cause error in `fleetd_logs` table time parsing
```

Comment:
```suggestion
* Hid "Self-service" in Fleet Desktop and My device page if there is no self-service software available.

* Fixed a bug that caused log Orbit's osquery table log output to be inconsistent.

* Added support for new agent option `script_execution_timeout` to configure seconds until a script is killed due to timeout.

* Updated Go version to go1.22.4.

* Fixed boot loop caused by Linux hosts with no hardware UUID.

* Added support for Linux ARM64.

* Fixed bug where UTC timezone could cause error in `fleetd_logs` table time parsing.
```

For consistency changelog items should be in the past-tense and always end with a period.

Reply:
Thanks! Changes committed. I'll add a reminder in the pull request template.